Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -124,102 +124,124 @@ export default class TimelineView extends Vue {
         this.filterText = this.filter.keyword;
     }
 
-    private immunizationLoadDeferred = false;
-    private immunizationLoadReady = false;
+    @Watch("immunizationIsDeferred")
+    private whenImmunizationIsDeferred(isDeferred: boolean) {
+        if (isDeferred) {
+            this.immunizationNeedsInput = true;
+        }
+    }
+
+    private get timelineEntries(): TimelineEntry[] {
```

Comment:
The entries are no longer stored in a stateful way inside the timeline; Instead, they are aggregated from the data sources when they change. This simplifies the logic of keeping the timeline updated when changes happen outside of it